bin/pycbc_live

```diff
@@ -996,6 +997,11 @@ parser.add_argument('--embright-massgap-max', type=float, default=5.0, metavar='
                          'HasMassGap probability.')
 parser.add_argument('--skymap-only-ifos', nargs='+',
                     help="Detectors that only contribute in sky localization")
+parser.add_argument('--psd-variation', action='store_true',
```

Comment:
Another way of 'turning on' the psd variation is to use the string provided to single ranking. 

I chose to do it this way to let psd var values be calculated and not used in the single ranking.

I think there does need to be a check within the parser args however to make sure that --psd-variation **is** given when somebody uses a single ranking that expects psd variation. Maybe this can be automatic?

Reply:
mmm, so in order to _use_ the psd variation we need to also change the single stat, that is what you are saying?  That makes sense .. 
I suppose if you do that but don't calculate psd var you get an error, have you checked if the error is comprehensible?

Reply:
"so in order to use the psd variation we need to also change the single stat" -> Yes, that's Arthur's alternative proposal. I prefer this way. One might want to compute the PSD variation, but not actually use it. No other statistic is controlled by the single ranking (e.g. chi-squared isn't turned off if we choose ranking=SNR).

Reply:
Sorry, I don't understand the difference here. It seems you are both saying the same thing

Reply:
> I suppose if you do that but don't calculate psd var you get an error, have you checked if the error is comprehensible?

From memory you would know you are missing psd_var values when calculating the statistic but you wouldn't know until a trigger came in. Which is why I would like to do an argument check to prevent running --sngl-stat '...psdvar' without having '--psd_variation' enabled.

Reply:
Well you can do that, no?

Reply:
Yeah, it'll be in the arg verification step. I just need to figure out how to do it 😁 will be a job for tomorrow